lib/utils/visFileH5.hpp

```diff
@@ -254,6 +254,10 @@ class visFileH5Fast : public visFileH5 {
 
 // These templated functions are needed in order to tell HighFive how the
 // various structs are converted into HDF5 datatypes
+const size_t DSET_ID_LEN = 33; // Length of the string used to represent dataset IDs
+struct dset_id_str {
+    char hash[DSET_ID_LEN];
```

Comment:
why does this need to be a char array inside a struct? Maybe @jrs65 can have a look at just the changes in this file

Reply:
it's to get around the limitations for writing fixed length strings through `HighFive`